### ISSUE TITLE
New Login Controls overlaps on Internet Explorer

### ISSUE THREAD

**Issue by paulobriennz** [volkanceylan/Serenity#973]
I like what you're flash new login page :)

Although on a Windows Remote Desktop connection over the web its a bit jittery.. be handy to know how to make it 'less interactive'..

I have just deployed latest build and the login controls are all overlapping on IE.
Looks OK on Chrome :/

**Comment by volkanceylan** [comment]
You could turn off animation, use a basic effect or use just one background image / none to make it less interactive. Settings are from Vegas library in LoginPanel.ts

What is IE version you are using? We no longer support < IE11, as only IE11 has proper flexbox layout.

**Comment by volkanceylan** [comment]
Hmm i see, it's also in IE11 :) Will have a look.

**Comment by paulobriennz** [comment]
thanks for advice re where to control animation.

I have tried from Windows 10 running IE 11.494.10586.0  and from a Windows 2012R2 Server thats pretty much patched up to date. 

![overlapping](https://cloud.githubusercontent.com/assets/19361762/17174373/2eb779e6-5455-11e6-957c-2fb09d30b1e7.PNG)

**Comment by jsbUSMC** [comment]
You can set the animation to not play the slideshow by adding `$('body').vegas('pause');` after initializing Vegas in your JavaScript at the end of the page on AccountLogin.cshtml.

**Comment by jsbUSMC** [comment]
Also can run that in console too. For production, remove that line for the slideshow to run.

**Comment by jsbUSMC** [comment]
Sorry I meant in your TypeScript file.

**Comment by volkanceylan** [comment]
Fixed look in 2.4.0.2

Need this css in your site.membership.less:

```css
.s-LoginPanel .flex-layout .s-PropertyGrid {
    display: block;
}
```

**Comment by paulobriennz** [comment]
thanks confirmed adding the css above fixed the issue.
